src/uwtools/tests/drivers/test_driver.py

```diff
@@ -35,40 +42,32 @@ def provisioned_run_directory(self):
     def _driver_name(self) -> str:
         return "concrete"
 
-    def _taskname(self, suffix: str) -> str:
-        return "concrete"
-
     def _validate(self) -> None:
         pass
 
-    @task
-    def atask(self):
-        yield "atask"
-        yield asset("atask", lambda: True)
 
+class ConcreteAssetsCycleBased(Common, driver.AssetsCycleBased):
+    pass
```

Comment:
There are now six "concrete" classes, each extending the new `Assets` and `Driver` intermediate classes and inheriting their methods from the `Common` class just above. Two of the classes aren't current used by tests, but are here for completeness and because we'll probably find a use for them before long.